roles/libvirt_manager/tasks/deploy_layout.yml

```diff
@@ -138,7 +138,7 @@
 
 - name: Dump MAC mapping
   ansible.builtin.copy:
-    dest: "{{ cifmw_libvirt_manager_basedir }}/artifacts/interfaces-info.yml"
+    dest: "{{ cifmw_libvirt_manager_basedir }}/artifacts/parameters/interfaces-info.yml"
```

Comment:
why adding that sublevel? Note it will break the reproducer.

Reply:
I was tracing back from the `reproducer` and noticed that every reference to `interfaces-info.yml` is from the `parameters` directory. This was the only odd place.

Reply:
Thank you for clarifying the reason why it was pointing to the current location